tests/wpt/metadata/html/browsers/windows/opener-closed.html.ini

```diff
@@ -0,0 +1,3 @@
+[opener-closed.html]
```

Comment:
Aha - this file should be in the `auxiliary-browsing-contexts` subdirectory of `browsers/windows/`. That's why it's not being picked up correctly.

Reply:
@jdm  Fixed it!